core/transaction/entity.go

```diff
@@ -105,6 +105,7 @@ const (
 	STORAGESC_WRITE_POOL_LOCK          = "write_pool_lock"
 	STORAGESC_WRITE_POOL_UNLOCK        = "write_pool_unlock"
 	STORAGESC_ADD_CURATOR              = "add_curator"
+	STORAGESC_REMOVE_CURATOR           = "remove_curator"
```

Comment:
Could we  use CamelCase instead of ALL_CAPS in Go names?

Reply:
Yes, but using the existing format takes preference.